cmd/deploy_test.go

```diff
@@ -14,45 +14,66 @@ import (
 	"github.com/Shopify/themekit/src/shopify"
 )
 
-func TestUpload(t *testing.T) {
+func TestUploadSingleFile(t *testing.T) {
```

Comment:
@tanema The existing test was getting pretty long, is it ok to split it up like this?

Reply:
Fixed in 7e0195a